ethcore/src/evm/evm.rs

```diff
@@ -95,11 +95,17 @@ impl<'a> Finalize for Result<GasLeft<'a>> {
 	}
 }
 
+/// Cost calculation type. For low-gas usage we calculate costs using usize instead of U256
```

Comment:
for 32-bit `usize`, this may lead to a consensus issue - `u64` should be a lot safer.